packages/twenty-chrome-extension/src/background/index.ts

```diff
@@ -1,4 +1,8 @@
+import Crypto from 'crypto-js';
```

Comment:
I cannot use `node:crypto` to create a `codeVerifier` and `codeChallenge` since this project is outside of node scope, hence the different alternative `crypto-js`